storage-operator/pkg/apis/storage/v1alpha1/volume_types.go

```diff
@@ -23,9 +23,18 @@ type RawBlockDeviceVolumeSource struct {
 	DevicePath string `json:"devicePath"`
 }
 
+type LVMLVSource struct {
+	// Name of the LVM VolumeGroup on the node to create the LVM LogicalVolume to back
+	// the PersistentVolume
+	VgName string `json:"vgName"`
+	// Size of the created LVM LogicalVolume backing the PersistentVolume
+	Size resource.Quantity `json:"size"`
+}
+
 type VolumeSource struct {
 	SparseLoopDevice *SparseLoopDeviceVolumeSource `json:"sparseLoopDevice,omitempty"`
 	RawBlockDevice   *RawBlockDeviceVolumeSource   `json:"rawBlockDevice,omitempty"`
+	LVMLV            *LVMLVSource                  `json:"LVMLV,omitempty"`
```

Comment:
Fully-capitalized looks a bit odd in a K8s object. Could we consider calling this `LogicalVolume` instead? Yeah theoretically there could be non-LVM 'logical volumes' on a system, but :man_shrugging:

Reply:
I agree it's odd :confused: 
Maybe `LVMLogicalVolume`, it's a bit long but it's the same as "SparseLoopDevice" so .... maybe better

Reply:
Discussed internally with @scality/metalk8s `LVMLogicalVolume` was chosen

Reply:
And in JSON, that'd then be `lvmLogicalVolume`?

Reply:
Hmm to me that will be more `LVMLogicalVolume` 4 capital at the beginning, but to me it shocks less.
But we can have `lvm` in minus but it's a bit odd to me

Reply:
It's fairly standard, field names aren't capitalized.

Reply:
Oh you right we talk about the name in the k8s object, you right let's have `lvmLogicalVolume`

Reply:
Heh, fine. Agree, in the Go struct it should be `LVMLogicalVolume`. I was wondering about the JSON serialization, where we then agree it's `lvmLogicalVolume`. All good :smile: